lib/goldiloader/active_record_patches.rb

```diff
@@ -47,7 +47,7 @@ def auto_include!(auto_include = true)
     end
 
     def auto_include_value
-      @values.fetch(:auto_include, true)
+      Goldiloader.enabled? && @values.fetch(:auto_include, true)
```

Comment:
Why was this change necessary? Isn't the change to `load_with_auto_include` sufficient?

Reply:
This is a remnant of the previous implementation. With the new change, it doesn't appear to be necessary. Removing.